tests/test_components.py

```diff
@@ -21,3 +32,45 @@ def test_metadata_default_values():
     metadata = Metadata()
     assert metadata.dateDownloaded is None
     assert metadata.probability is None
+
+
+def test_anchor_optional_fields():
+    Link(text="foo")
+    Link(url="https://example.com")
+
+
+def test_business_place_metadata_default_values():
+    metadata = BusinessPlaceMetadata()
+    assert metadata.dateDownloaded is None
+    assert metadata.probability is None
+    assert metadata.searchText is None
+
+
+def test_address():
+    Address(addressRaw="2440 Hoonani Rd, Koloa, HI 96756, US")
+    Address(streetAddress="2440 Hoonani Rd, Koloa, HI 96756")
+    Address(addressCity="Koloa")
+    Address(addressLocality="Poipu")
+    Address(addressRegion="Kauaʻi County")
+    Address(addressCountry="US")
+    Address(postalCode="HI 96756")
```

Comment:
```suggestion
    Address(postalCode="96756")
```

We also seem to be missing `postalCodeAux="HI"`.